tests/unit/sagemaker/model/test_model_package.py

```diff
@@ -52,6 +52,10 @@
     "CertifyForMarketplace": False,
 }
 
+ENV_KEY_1 = "env_key_1"
+ENV_VALUE_1 = "env_key_1"
+ENVIRONMENT = {ENV_KEY_1: ENV_VALUE_1}
+
```

Comment:
If these 3 variables are only used in test_create_sagemaker_model_include_environment_variable function, then the scope can be local to the function itself, hence we can move these constants within the function itself.

Reply:
updating